boards/cc3200-launchxl/include/periph_conf.h

```diff
@@ -0,0 +1,115 @@
+/*
+ * Copyright (C) 2019 Ludwig Maximilian Universität
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     boards_cc3200-launchxl
+ * @{
+ *
+ * @file
+ * @brief       Peripheral MCU configuration for the CC3200 board
+ *
+ * @author      Wladislaw Meixner <wladislaw.meixner@campus.lmu.de>
+ */
+
+#ifndef PERIPH_CONF_H_
+#define PERIPH_CONF_H_
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @name Timer peripheral configuration
+ *
+ */
+
+#include "periph_cpu.h"
+
+#define CLOCK_CORECLOCK (80000000U) /* desired core clock frequency */
+
+/*  CC3200 has 4 timer blocks */
+#define TIMER_0_EN 0
```

Comment:
Shouldn't it be `1` if it's enabled?

I see it's used as an index for the rest of the code, but `TIMER_0_EN` also has a meaning for [dev_enums.h](https://github.com/RIOT-OS/RIOT/blob/master/drivers/include/periph/dev_enums.h#L36).

Reply:
Okay I see. I will just replace the enum usage with some constants, the enum is just used for the interrupt handler functions.